code/modules/hydroponics/hydroponics.dm

```diff
@@ -135,6 +136,10 @@
 		mutate()
 	else if(istype(Proj , /obj/projectile/energy/florayield))
 		return myseed.bullet_act(Proj)
+	else if(istype(Proj , /obj/projectile/energy/florarevolution))
+		mutatespecie()
+		if(myseed)
+			myseed.instability = (myseed.instability/2)

```

Comment:
Is it intended that you can use the floral revolution feature to quickly decrease the instability of plants that don't have any possible species mutations? IMO, this is fine, but I just wanted to make you aware of that trick.